changelog.d/15613.feature

```diff
@@ -0,0 +1 @@
+Flake.nix: `devenv up` now automatically starts synapse, configured to use redis and postgresql.
```

Comment:
likewise:

```suggestion
Running `devenv up` in the nix Synapse development environment now automatically starts and configures Synapse to use Redis and PostgreSQL.
```

Reply:
Since this is only for development I think I'd just axe this changelog.

Reply:
Ok thanks @clokep. Done.